packages/transaction-controller/src/TransactionController.ts

```diff
@@ -755,7 +755,7 @@ export class TransactionController extends BaseController<
         )) ||
       (existingMaxPriorityFeePerGas && minMaxPriorityFeePerGas);
 
-    const txParams: TransactionParams =
+    const cancelTxParams: TransactionParams =
```

Comment:
Shall we call this `newTxParams` for clarity?

Reply:
Renamed in https://github.com/MetaMask/core/pull/1998/commits/0ee039e86a8ff7ab397943748f5103e0e6e5caba